lib/pdf_fill/forms/va210781a.rb

```diff
@@ -216,6 +225,15 @@ def merge_fields
 
       private
 
+      def expand_other_information
+        return if @form_data['otherInformation'].blank?
+        other_information_lines = []
+        @form_data['otherInformation'].each do |other_information|
+          other_information_lines.push('value' => other_information)
+        end
```

Comment:
Could this be replaced with something like this?
```
@form_data['otherInformation'].map! do |other_info|
  'value' => other_info
end
```

Reply:
Yes, can do!